.github/CODEOWNERS

```diff
@@ -61,6 +61,8 @@
 /include/swift/DependencyScan                      @artemcm
 /include/swift/Driver                              @artemcm
 # TODO: /include/swift/IRGen/
+/include/swift/IDE/                                @ahoppen @rintaro
+/include/swift/Refactoring                         @ahoppen @bnbarham
```

Comment:
I see you’ve added `test/Index`. Were the indexing sources purposely omitted?

Reply:
I just missed them. Thanks for noticing